x-pack/packages/kbn-elastic-assistant-common/impl/capabilities/index.ts

```diff
@@ -19,6 +19,7 @@ export type AssistantFeatureKey = keyof AssistantFeatures;
  * Default features available to the elastic assistant
  */
 export const defaultAssistantFeatures = Object.freeze({
-  assistantKnowledgeBaseByDefault: false,
+  assistantKnowledgeBaseByDefault: true,
```

Comment:
This should remain false per @spong

Reply:
Yes yes, thanks for the ping @stephmilovic! Since only half of the KB work ended up landing for `8.15` this flag is still in use, and should be set to `false` here in code. I will rename/update the comment in an upcoming PR to clear up confusion here.